plugins/chat/assets/javascripts/discourse/connectors/user-profile-controls/chat-button.gjs

```diff
@@ -0,0 +1,11 @@
+import ChatDirectMessageButton from "../../components/chat/direct-message-button";
+
+const ChatButton = <template>
+  {{#if @outletArgs.model.can_chat_user}}
+    <li class="user-card-below-message-button chat-button">
```

Comment:
I don't understand why these changes when converting from gjs... these are being auto-injected for hbs but not for gjs?

Reply:
For plugin-outlet connectors, yeah. If discourse sees a raw .hbs file, we render it with the PluginConnector classic component class, which has these wrapper elements.